src/btree/bt_handle.c

```diff
@@ -32,7 +32,7 @@ __wt_btree_page_version_config(WT_SESSION_IMPL *session)
  * values may require update when the release is named.
  */
 #define WT_VERSION_TS_MAJOR 3
-#define WT_VERSION_TS_MINOR 3
+#define WT_VERSION_TS_MINOR 4
```

Comment:
This deserves a comment, and I think we should pick a release that exists (I know I said otherwise in slack, sorry). Could this change to 10.0 instead of 3.4?

Reply:
I updated the comment and version numbers.